quickstart/README.md

```diff
@@ -3,6 +3,39 @@
 This directory contains configuration files for `control-plane` and `piped` for [Quickstart](https://pipecd.dev/docs/quickstart) Guide.
 
 
+
+### Quickstart with raw manifests
+
+1. create namespace `pipecd`
+
+```
+kubectl create namespace pipecd
+```
+
+2. deploy Control Plane to the namespace: pipecd
+
+```
+kubectl apply -n pipecd -f ./manifests/control-plane.yaml
+```
+
+3. create piped on the Web UI
```

Comment:
I think it's nice for users to show kubectl port-forward command and URL of the Web UI.
WDYT?

Reply:
@Warashi 
It's a nice idea. But I wonder if I should do so 🤔 
For me, this doc is like a memo to describe whats the manifest is. 
Also there is the actual process of the quickstart on the docs. https://pipecd.dev/docs-v0.48.x/quickstart/

Reply:
OK, I got it. I agree that we have other documents about these manifests and don't have to show detailed commands here.